examples/windows/windows-perfmon-proc-owner.ps1

```diff
@@ -0,0 +1,44 @@
+$computername = "DESKTOP-26GUTUU"
+$colProcs = Get-wmiobject win32_process -computername $computername  -Credential $credential | select *, @{Name = "Owner"; Expression = { ($_.GetOwner()).User } }
+$colPerfs = Get-wmiobject win32_perfformatteddata_perfproc_process -computername $computername  -Credential $credential
+$colTasklist = @()
+ 
+foreach ($perf in $colPerfs) {
+    if ($perf.Name.StartsWith("w3wp")) {
```

Comment:
filtering should be pushed into the `Get-CimInstance` currently `get-wmiobject` to reduce query load

Reply:
fixed